### PR TITLE
feat: added prepo-io/stores to prepo-stores monorepo

### PR DESCRIPTION
**Pull request by georgesmithishere**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=1f519ba1849347aa9740faedda4acf3b&p=18614b05af154f39ac3bed844f6ccb3b

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>react-boilerplate</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://react-boilerplate-3slphmkdu-prepo.vercel.app'>https://react-boilerplate-3slphmkdu-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_3Lydru5gdN5RuqWFQBT8AZrbiROq"></hidden>

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-gk97cv6s9-prepo.vercel.app'>https://core-dapp-gk97cv6s9-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

File: packages/prepo-stores/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# @prepo-io/stores
```
Comment (r0x4xn) [inline packages/prepo-stores/README.md:1]: rename this one, it's now called `prepo-stores`

File: packages/prepo-stores/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+yarn link @prepo-io/stores
+```
+
+## Troubleshooting
+
+You might need to run in this project if you get "Invalid hook call" error with missmatched React versions.
+
+```bash
+npm link ../<project-name>/node_modules/react
+```
```
Comment (r0x4xn) [inline packages/prepo-stores/README.md:43]: All the content on the readme is not relevant anymore

File: packages/prepo-stores/src/ContractStore.ts [hunk 3]
```diff
@@ -0,0 +1,222 @@
+import { ContractCallContext } from 'ethereum-multicall'
+import { action, autorun, makeObservable, observable, onBecomeUnobserved, runInAction } from 'mobx'
+import { Contract, ContractFunction, BigNumber, UnsignedTransaction } from 'ethers'
+// import { SupportedContractName } from 'prepo-constants'
```
Comment (r0x4xn) [inline packages/prepo-stores/src/ContractStore.ts:4]: We can remove this comment if we are not using this

**Review by r0x4xn — DISMISSED** [review]
CHANGELOG.md should not be migrated

File: packages/prepo-stores/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# @prepo-stores
```
Comment (chrisling-dev) [inline packages/prepo-stores/README.md:1]: Let's name this consistently across different packages
```suggestion
# prePO Stores package
```

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
Nice, just 2 comments

**Review by chrisling-dev — APPROVED** [review]

**Pull request merged**